sections/02_intro.md

```diff
@@ -1,5 +1,51 @@
 ## Introduction
 
+Biology and medicine are rapidly becoming particularly data-intensive field with
+respect to both research and practice. A recent comparison of genomics with
+social media, online videos and other data-intensive scientific disciplines
+suggested that the field of genomics alone would equal or surpass other fields
+in data generation and analysis within the next decade
+[@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
+also new challenges. We expect that algorithms to automatically extract
+meaningful patterns and provide sufficient context to enable us to act will be
+required.
+
+Concurrent with this explosive growth in biomedical data, a new class of machine
+learning algorithm has become widespread in the domain of image analysis.
+Computer scientists are now building many-layered neural networks from
+collections of millions of images. In a famous example, scientists from Google
+demonstrated that a neural network could learn to identify cats simply by
+watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
+termed deep learning, seem like a solution to the challenge presented by the
+growth of data in biomedicine. Perhaps these algorithms could identify the
+biological "cats" hidden in our data - the patterns that exist but that we don't
+know to look for - and could act on them.
+
+Deep learning has transformed image analysis, but researchers' initial forays
+into the use of these techniques in biomedicine have been relatively limited.
+There are certainly numerous promising examples, but we have not yet seen the
+massive convergence on these approaches that occurred in the field of image
+analysis. In this review, we discuss whether this is simply a matter of time or
+if there are unique challenges posed by biomedical data that render deep
+learning methods more challenging or less fruitful.
+
+`TODO: not sure if it should go here, but somewhere we should talk about how we
```

Comment:
I agree this is interesting to discuss in the review, but I suggest it come later in the paper.  Perhaps in a non-traditional Author Contributions section?  I think it is distracting in the intro when we are trying to engage readers.